components/calendar/src/chinese_based.rs

```diff
@@ -154,13 +152,9 @@ impl<C: ChineseBased + CalendarArithmetic> ChineseBasedDateInner<C> {
         let month_after_twelfth =
             Self::new_moon_on_or_after((month_after_eleventh + 1).as_moment()); // m13
         let next_eleventh_month = Self::new_moon_before((following_solstice + 1).as_moment()); // next-m11
-        let lhs_argument = libm::round(
-            div_rem_euclid_f64(
-                (next_eleventh_month - month_after_eleventh) as f64,
-                MEAN_SYNODIC_MONTH,
-            )
-            .0,
-        ) as i64;
+        let lhs_argument =
+            libm::round((next_eleventh_month - month_after_eleventh) as f64 / MEAN_SYNODIC_MONTH)
+                as i64;
```

Comment:
Question: Why did you change this from divrem euclid to normal div?

Reply:
`next_eleventh_month` should always be greater than `month_after_eleventh`, so there should never be a difference between using `div_rem_euclid` and normal `/`. It's just a little simpler, but there's really no difference, so if you want I can change it back